3rdparty/snmalloc/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ add_custom_command(
   DEPENDS musl_includes libcxx_includes)
 
 add_enclave_library(
-  oesnmalloc
+  oesnmalloc_obj
```

Comment:
It would be good to add comments explaining why we need both `oesnmalloc_obj` and `oesnmalloc`.

Reply:
Done.